src/Nethermind/Nethermind.Consensus.AuRa/Validators/ContractBasedValidator.Posdao.cs

```diff
@@ -42,7 +43,7 @@ public IEnumerable<Transaction> GetTransactions(BlockHeader parent, long gasLimi
                     {
                         emitInitChangeCallable = ValidatorContract.EmitInitiateChangeCallable(parent);
                     }
-                    catch (AuRaException e)
+                    catch (AbiException e)
```

Comment:
you change AbiException to AuRa and  you changed the exception that you catch from AuRa to Abi. Was it intended?